cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -107,10 +117,35 @@ def test_order_catalog_item(catalog_item, request):
     catalog_item.create()
     service_catalogs = ServiceCatalogs(catalog_item.name)
     service_catalogs.order()
-    logger.info('Waiting for cfme provision request for service %s', catalog_item.name)
-    row_description = catalog_item.name
-    cells = {'Description': row_description}
+    logger.info('Waiting for cfme provision request for service {}'.format(catalog_item.name))
```

Comment:
I think you may have missed my update to my comment. Milan corrected me, for logging we do want to use %s, because logger takes the argument.